packages/happtiq_commons_google_cloud/src/happtiq_commons_google_cloud/gcs_api_service.py

```diff
@@ -37,3 +37,13 @@ def upload_file(self, file_path: bytes,  bucket_name: str, destination: str, con
         blob.upload_from_filename(file_path, content_type=content_type, timeout=self.timeout)
         self.logger.info(f"File uploaded to {destination}.")
         return f"gs://{bucket_name}/{destination}"
+    
+    def list_files_with_prefix(self, bucket_name: str, prefix: str):
+        self.logger.info(f"Listing files in bucket {bucket_name} with prefix {prefix}")
+        bucket = self.client.bucket(bucket_name)
+        blobs = bucket.list_blobs(prefix=prefix)
+
+        file_list = [blob.name for blob in blobs]
+        self.logger.info(f"Files in bucket {bucket_name} with prefix {prefix}: {file_list}")
```

Comment:
change this to a debug log and add another info log "found {len(file_list)} files in bucket {bucket_name} with prefix {prefix}

Reply:
I added the debug log and updated the info log. I will push the changes to the branch, please check